inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/sidebar/ActiveLearningSidebar.html

```diff
@@ -32,20 +32,27 @@ <h3 class="panel-title">Session</h3>
             <div class="form-group form-horizontal">
               <label class="col-sm-3 control-label">Layer</label>
               <div class="col-sm-9">
-                <select class="form-control"  wicket:id="selectLayer"></select>
+                <select class="form-control" wicket:id="selectLayer"></select>
                 <span class="form-control" wicket:id="showSelectedLayer"></span>
               </div>
             </div>
           </div>
           <div class="panel-footer text-right">
-            <input wicket:id="layerSelectionButton" type="submit" class="btn btn-primary" value="Start"/>
+            <input wicket:id="layerSelectionButton" type="submit" class="btn btn-primary"
```

Comment:
Usually we use a `<button type"submit"...>` in stead of an `input`. But practically, it probably does not matter.